packages/api/src/routers/board.ts

```diff
@@ -352,4 +352,37 @@ export const boardRouter = createTRPCRouter({
 
       return { success: true };
     }),
+    checkSlugAvailability: publicProcedure
+      .meta({
+        openapi: {
+          summary: "Check if a board slug is available",
+          method: "GET",
+          path: "/boards/check-slug-availability",
```

Comment:
I think it makes more sense to pass in the boardPublicId

```suggestion
          path: "/boards/{boardPublicId}/check-slug-availability",
```